docs/multi-user.md

```diff
@@ -45,6 +45,79 @@ If we want to give access to multiple namespaces, we can use a single cluster ro
 
 This makes sure, tenants can't interfere with GitRepo resources from other tenants, since they don't have access to their namespaces.
 
+## Example Fleet in Rancher
+ 
+When a new fleet workspace is created, a corresponding namespace with an identical name is automatically generated within the Rancher local cluster.
+For a user to see and deploy fleet resources in a specific workspace needs at least the following permissions:
```

Comment:
Are we talking about permissions that the _user_ must have? If so, I find this a bit clearer:
```suggestion
For a user to see and deploy fleet resources in a specific workspace, they need at least the following permissions:
```